bundles/core/src/main/java/com/adobe/cq/commerce/core/components/models/teaser/CommerceTeaser.java

```diff
@@ -23,16 +23,14 @@
 public interface CommerceTeaser extends Teaser {
 
     /**
-     * Name of the resource property that stores the Product SKU which is used to build Call-to-Action link for Product Page
+     * Name of the resource property that stores the Product Slug which is used to build Call-to-Action link for Product Page
      *
-     * @since com.adobe.cq.wcm.core.components.models 12.4.0
      */
-    String PN_ACTION_PRODUCT_SKU = "productSKU";
+    String PN_ACTION_PRODUCT_SLUG = "productSlug";
 
     /**
-     * Name of the resource property that stores Category ID to which is used to build Call-to-Action link for Category Page
+     * Name of the resource property that stores Category Slug which is used to build Call-to-Action link for Category Page
```

Comment:
please revert

Reply:
aah,, you mean fix the typo `category ID to` -> `category ID` , i.e. removing the `to` , or README.md to refer as `...stores Category Slug...` ?

Reply:
I meant "revert your change renaming ID to Slug", but yes you can also remove the unneeded "to"

to have "Name of the resource property that stores Category ID which"

Reply:
> Name of the resource property that stores Category ID which is used to build Call-to-Action link for Category Page

Reply:
got it, done